test/framework/exec/kubectl.go

```diff
@@ -20,16 +20,21 @@ import (
 	"bytes"
 	"context"
 	"fmt"
+	"os"
 )
 
 // KubectlApply shells out to kubectl apply.
 //
 // TODO: Remove this usage of kubectl and replace with a function from apply.go using the controller-runtime client.
 func KubectlApply(ctx context.Context, kubeconfigPath string, resources []byte, args ...string) error {
+	kubectlPath, ok := os.LookupEnv("CAPI_KUBECTL_PATH")
+	if !ok {
+		kubectlPath = "kubectl"
+	}
```

Comment:
nit: let's create `func kubectlPath() string` instead of duplicating this three lines in different places

Reply:
thanks, fixed